src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,92 +15,111 @@ redirects:
 
-## View host data in APM [#infra-apm-overview]
+You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
 
-When your <InlinePopover type="apm" /> and infrastructure data is linked, you have more available host data in APM.
+## Filter by application data [#filter-by-app]
 
-The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page contains a table with data about your app's hosts and instances, including:
+When your <InlinePopover type="apm" /> and infrastructure data is linked, you can filter displayed host data by searching for the specific application you want to inspect. In the case above, you would want to filter for `Billing Service`.
 
-* Apdex
-* Response time
-* Throughput
-* Error rate
-* CPU usage
-* Memory
+<img
+    title="filter by application"
+    alt="A screenshot showing you how to filter the infrastructure page by application"
+    src={apmFilterbyApplication}
+/>
 
-You can toggle between a table view or breakout metric details for the individual hosts by selecting **View table** or **Break out each metric by host**.
+## APM data on inventory and events UI pages [#inventory-events]
 
-For more information on host data on the APM **Summary** page, see [host details](/docs/apm/applications-menu/monitoring/apm-overview-page#host_details).
+When your <InlinePopover type="apm" /> and infrastructure data is linked, you can view and filter on application data on the Infrastructure monitoring UI's [**Inventory** page](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure) and the [**Events** page](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-events-page-live-feed-every-agent).
 
 ## Troubleshoot missing APM data [#troubleshooting]
 
-APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set. If you do not see APM data in infrastructure monitoring, see [Troubleshooting](/docs/infrastructure/new-relic-infrastructure/troubleshooting/troubleshooting-apm-data-not-infrastructure).
+APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set. 
```

Comment:
The slash feels a bit awkward here, maybe just APM and Infrastructure?

Reply:
There are 2 ands at the end of this section, I think it should be "agent installed on the same hosts, they use the same license key, and have the same hostname set.